src/main/java/org/thingsboard/gateway/extensions/http/HttpController.java

```diff
@@ -22,29 +22,28 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.web.bind.annotation.*;
-import org.thingsboard.gateway.extensions.sigfox.conf.SigfoxRequestProcessingError;
+import org.thingsboard.gateway.extensions.http.conf.HttpRequestProcessingError;
 
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
 
 @RestController
-@RequestMapping("/sigfox")
-@ConditionalOnProperty(prefix = "sigfox", value = "enabled", havingValue = "true")
+@ConditionalOnProperty(prefix = "http", value = "enabled", havingValue = "true")
 @Slf4j
-public class SigfoxController {
+public class HttpController {
     private static final String TOKEN_HEADER = "Authorization";
 
     @Autowired
-    private SigfoxService service;
+    private HttpService service;
 
     private ObjectMapper mapper = new ObjectMapper();
 
 
-    @RequestMapping(value = "/{deviceTypeId}", method = RequestMethod.POST)
-    public void handleRequest(@PathVariable String deviceTypeId,
+    @RequestMapping(value = "/{baseUrl}/{converterId}", method = RequestMethod.POST)
```

Comment:
Please add two similar methods, or two mappings 
@RequestMapping(value = "/sigfox/{deviceTypeId}", method = RequestMethod.POST)
@RequestMapping(value = "/uplink/{converterId}", method = RequestMethod.POST)

Reply:
okey, will be fixed.